.github/workflows/deploy-to-control-plane-staging.yml

```
@@ -1,32 +1,94 @@
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Setup Environment
+        uses: ./.github/actions/setup-environment
+        with:
+          token: ${{ secrets.CPLN_TOKEN_STAGING }}
+          org: ${{ vars.CPLN_ORG_STAGING }}
 
-      - uses: ./.github/actions/deploy-to-control-plane
+      - name: Deploy to Control Plane
+        uses: ./.github/actions/deploy-to-control-plane
```

actionlint (1.7.4)

missing input "pr_number" which is required by action "Deploy to Control Plane" defined at "./.github/actions/deploy-to-control-plane". all required inputs are "app_name", "cpln_token", "github_token", "org", "pr_number"

(action)